lib/FIR/afforestation.cc

```diff
@@ -1317,84 +1337,150 @@ struct FortranIRLowering {
   // CALL translations ...
   const Value *CreateCalleeValue(
       const parser::ProcedureDesignator &designator) {
-    return static_cast<const Value *>(nullptr);
+    return nullptr;
   }
   CallArguments CreateCallArguments(
       const std::list<parser::ActualArgSpec> &arguments) {
     return CallArguments{};
   }
 
+  Expression VariableToExpression(const parser::Variable &var) {
+    auto maybe{semantics::AnalyzeVariable(semanticsContext_, var)};
+    return {std::move(*maybe)};
```

Comment:
Can't `maybe` be `std::nullopt`?

Reply:
It shouldn't be (I don't think), but I can add a `CHECK` to get a nicer looking crater message.

Reply:
Or use `maybe.value()`.

Reply:
done